pages/project/add-user/content/index.js

```diff
@@ -0,0 +1,15 @@
+const { merge } = require('lodash');
+const baseContent = require('../../content');
+
+module.exports = merge({}, baseContent, {
+  title: 'Grant access',
+  fields: {
+    profile: {
+      label: 'Who needs to be able to view this project?',
+      hint: 'If the person you’re looking for doesn’t appear in the list, ask a [HOLC at your establishment]({{profilesHolcFiltered}}) to give them access on your behalf.'
```

Comment:
The link is a nice touch.

Reply:
Except that it should probably be "admin" rather than HOLC, no?

Reply:
I've asked the question - but you are probably right